docker/generated/model-server/3.6/alpine3.14/Dockerfile

```diff
@@ -0,0 +1,60 @@
+# syntax = docker/dockerfile:experimental
```

Comment:
enable developer to quickly build image with `ENABLE_BUILDKIT=1`. Othewise we will take advantage of `docker.api.build` from docker-py